run-oft.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
set -euo pipefail
```
Is a bash available in the docker image?
I created https://github.com/itsallcode/openfasttrace-github-action/issues/2 to run a linter for the shell script. Not necessary in this PR.

Reply:
no, it is not (by default). We could install it but I do not really see the benefit, do you?

Reply:
No, you are right, there is no benefit.